core/settings.py

```diff
@@ -48,8 +48,6 @@
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 # https://docs.djangoproject.com/en/3.1/ref/settings#s-secure-ssl-redirect
 SECURE_SSL_REDIRECT = False  # SSL redirect is handled by Nginx reverse proxy in prod.
-# https://docs.djangoproject.com/en/dev/ref/settings/#session-cookie-httponly
-SESSION_COOKIE_HTTPONLY = IS_PROD
```

Comment:
SESSION_COOKIE_HTTPONLY is True by default, and as experience has shown, it's best for us to have the same setting in dev environment as in prod (so that we aren't surprised by the different behavior after deploying).